ethcore/engines/authority-round/src/lib.rs

```diff
@@ -735,10 +734,10 @@ fn verify_timestamp(step: &Step, header_step: u64) -> Result<(), BlockError> {
 			// Returning it further won't recover the sync process.
 			trace!(target: "engine", "verify_timestamp: block too early");
 
-			let found = CheckedSystemTime::checked_add(UNIX_EPOCH, Duration::from_secs(oob.found))
+			let found = UNIX_EPOCH.checked_add(Duration::from_secs(oob.found))
 				.ok_or(BlockError::TimestampOverflow)?;
-			let max = oob.max.and_then(|m| CheckedSystemTime::checked_add(UNIX_EPOCH, Duration::from_secs(m)));
-			let min = oob.min.and_then(|m| CheckedSystemTime::checked_add(UNIX_EPOCH, Duration::from_secs(m)));
+			let max = oob.max.and_then(|m| UNIX_EPOCH.checked_add(Duration::from_secs(m)));
+			let min = oob.min.and_then(|m| UNIX_EPOCH.checked_add(Duration::from_secs(m)));
```

Comment:
for authority-round the issue of `SystemTime` being platform dependant doesn't really matter, since we use it in the error path here anyway.